dll/win32/shell32/lang/bg-BG.rc

```diff
@@ -1003,4 +1003,8 @@ BEGIN
     IDS_COPYTOTITLE "Select the place where you want to copy '%s'. Then click the Copy button."
     IDS_COPYITEMS "Copy Items"
     IDS_COPYBUTTON "Copy"
+    IDS_MOVETOMENU "Move to &folder..."
+    IDS_MOVETOTITLE "Select the place where you want to move '%s' to. Then click the Copy button."
```

Comment:
This line is wrong in all rc language files except the german one IDS_MOVETOTITLE "Then click the Copy button." should be "Then click the Move button." for all languages!

Reply:
Ah good. You fixed that already! Nice!

Reply:
OK.